models/FollowModel.js

```diff
@@ -0,0 +1,24 @@
+const followUser = `INSERT INTO follow ("followerUserId", "followedUserId")
+SELECT $1, "userId"
+FROM app_user
+WHERE "userId" = $2;
+`;
```

Comment:
I think this can be simplified into
```suggestion
const followUser = `INSERT INTO follow ("followerUserId", "followedUserId")
VALUES ($1, $2)`;
```